DSharpPlus/Entities/DiscordVoiceRegion.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using Newtonsoft.Json;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
Did you get intellisense'd?

Reply:
lmao yes I did but I thought I removed that.